### ISSUE TITLE
Roll back ALL stacks in the selected stack pattern

### ISSUE THREAD

**Issue by rehanvdm** [rehanvdm/cdk-express-pipeline#4]
Read the pattern `cdk deploy `{pattern}` that was used and then if one of the stacks fails for that pattern, roll all of them back. This might involve injecting a custom resource and a Cloudwatch trigger to be monitored, then if one fails, fail all of them. The concurrency will have to be set high enough so that all stacks in the pattern are deployed at once. Just an idea, look for alternative solutions before starting.

**Comment by rehanvdm** [comment]
A lot of worked, currently parked on branch: https://github.com/rehanvdm/cdk-express-pipeline/compare/poc...main
